elements/lisk-cryptography/src/address.ts

```diff
@@ -138,19 +137,17 @@ export const validateLisk32Address = (
 
 	const addressSubstringArray = address.substring(3).split('');
 
-	if (!addressSubstringArray.every(char => LISK32_CHARSET.includes(char))) {
+	if (!addressSubstringArray.every(char => BASE32_CHARSET.includes(char))) {
 		throw new Error(
-			"Invalid character found in address. Only allow characters: 'abcdefghjkmnopqrstuvwxyz23456789'.",
+			`Invalid character found in address. Only allow characters: '${BASE32_CHARSET}'.`,
 		);
 	}
 
-	const integerSequence = addressSubstringArray.map(char => LISK32_CHARSET.indexOf(char));
+	const integerSequence = addressSubstringArray.map(char => BASE32_CHARSET.indexOf(char));
 
 	if (!verifyChecksum(integerSequence)) {
 		throw new Error('Invalid checksum for address.');
 	}
-
-	return true;
```

Comment:
this is breaking change, and it might cause the other tools to fail

Reply:
I personally find it quite inconsistent that a function throws when validation fails, returns true when everything validation succeeds, and never returns false. But I have reverted to the previous state.